optional-kubernetes-engine/Gemfile

```diff
@@ -19,17 +19,15 @@ gem "fog"
 gem "omniauth"
 gem "omniauth-google-oauth2"
 # [START gcloud]
-gem "gcloud", "~> 0.8.0"
+gem "google-cloud"
```

Comment:
Can we set a version spec for this? `"~> 0.50"`

Reply:
Of course, I'll update the Gemfile to use a pessimistic versioning. Thanks Daniel!